account_move_helper/models/account_account.py

```diff
@@ -33,13 +31,17 @@ def _compute_new_balance(self):
 
     @api.multi
     def _compute_balance(self):
```

Comment:
* use rec.env['account.move.line'] as variable out of the loop in order to re use.
* use rec.update() instead of rec.field =